src/utils/export-map.ts

```diff
@@ -161,7 +193,13 @@ export class ExportMap {
   }
 
   static get(source: string, context: RuleContext) {
-    const path = resolve(source, context)
+    const tsconfig = lazy(() => getTsconfigWithContext(context))
+
+    const path = resolve(source, context, {
+      get tsconfig() {
+        return tsconfig()
+      },
+    })
```

Comment:
If this is already a getter, just use the `defineLazyProperty` util.

Reply:
Isn't the same thing?

Reply:
You mean `defineLazyProperty({}, 'tsconfig', taconfig)`?